src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/fullScript_approved.properties

```diff
@@ -0,0 +1,63 @@
+approvedScripts=Approved scripts
+
+# approved scripts
+metadataGatheringDisabled=The metadata gathering was disabled. \
+  <p>You can re-enable it permanently by removing the system property: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.metadataGathering</code> </p>\
+  <p>or temporarily (until restart) by running this in the Script Console: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.METADATA_GATHERING = true</code></p>
```

Comment:
I don't understand this.

What disabled it? (was it the preceding user action?) - in which case, I'd probably drop `was`.
Naively, I'd change `The metadata` to `Metadata`.

Reply:
By default it's enabled but you can disabled it by either using the system property or running the "reverse" script.

Changing to: "Metadata gathering disabled."